examples/chart/teleport-relay/tests/deployment_test.yaml

```diff
@@ -0,0 +1,17 @@
+suite: Test Deployment
+templates:
+  - deployment.yaml
+  - configmap.yaml
+tests:
+  - it: should match the snapshot (simple)
+    template: deployment.yaml
+    values:
+      - ../.lint/simple.yaml
+    asserts:
+      - matchSnapshot: {}
+  - it: should match the snapshot (full)
+    template: deployment.yaml
+    values:
+      - ../.lint/full.yaml
+    asserts:
+      - matchSnapshot: {}
```

Comment:
I really don't like those snapshot tests, they provide little value and cause an constant noise when we update the release branch as the version and labels change.

Snapshots are useful on things that cannot be tested otherwise (e.g. config content) but here, deployment features can be tested.

Reply:
Redone the tests to specifically test just about every tricky thing we do.